shipgen/EvtCalcGenerator.cxx

```diff
@@ -0,0 +1,121 @@
+#include <math.h>
+#include "TROOT.h"
+#include "TFile.h"
+#include "FairPrimaryGenerator.h"
+#include "EvtCalcGenerator.h"
+#include "TDatabasePDG.h"
+#include "TMath.h"
+#include "TGeoVolume.h"
+#include "TGeoNode.h"
+#include "TGeoManager.h"
+#include "TGeoEltu.h"
+#include "TVectorD.h"
+#include "TGeoCompositeShape.h"
```

Comment:
I find it hard to believe that all of these are used.

Reply:
Cleanup implemented.